README.md

```diff
@@ -69,6 +69,7 @@ The following checks are performed when calling the binary:
 - `pkgs/by-name` must only contain subdirectories of the form `${shard}/${name}`, called _package directories_.
 - The `name`'s of package directories must be unique when lowercased.
 - `name` is a string only consisting of the ASCII characters `a-z`, `A-Z`, `0-9`, `-` or `_`.
+- `name` only starts with an ASCII character `a-z`, `A-Z`, `-` or `_`.
```

Comment:
```suggestion
- `name` only starts with an ASCII character `a-z`, `A-Z` or `_`.
```

It cannot start with `-`. `-` will be parsed to a negation operator.

Reply:
I don't think this needs to be done in this same PR, since it's technically a separate problem, but yeah this would be good too :)

Reply:
> I don't think this needs to be done in this same PR, since it's technically a separate problem, but yeah this would be good too :)

It's not allowed in the regex already.